slither/solc_parsing/declarations/contract.py

```diff
@@ -574,21 +574,26 @@ def analyze_state_variables(self):
         except (VariableNotFound, KeyError) as e:
             self.log_incorrect_parsing(f"Missing state variable {e}")
 
-    def analyze_using_for(self):
+    def analyze_using_for(self):  # pylint: disable=too-many-branches
         try:
             for father in self._contract.inheritance:
                 self._contract.using_for.update(father.using_for)
```

Comment:
@smonicas One thing I wasn't sure about is if there'd be a reason we also need to update child contracts' `using_for_complete`

Reply:
I don't think there is a need because the first use of `using_for_complete` is when converting to slithir which comes after parsing is finished